examples.md

```diff
@@ -108,3 +109,14 @@ uses: actions/cache@preview
     restore-keys: |
       ${{ runner.os }}-go-
 ```
+
+## Python - pip
+
+```yaml
+- uses: actions/cache@preview
+  with:
+    path: ~/.cache/pip
```

Comment:
This only works for Linux, see https://github.com/actions/cache/issues/3#issuecomment-548436296.

Reply:
Oh.. thank you for your advice :)